code/modules/tgui/modules/ert_manager.dm

```diff
@@ -0,0 +1,100 @@
+/datum/tgui_module/ert_manager
+	name = "ERT Manager"
+	var/ert_type = "Red"
+	var/commander_slots = 1 // defaults for open slots
+	var/security_slots = 4
+	var/medical_slots = 0
+	var/engineering_slots = 0
+	var/janitor_slots = 0
+	var/paranormal_slots = 0
+	var/cyborg_slots = 0
+
+/datum/tgui_module/ert_manager/tgui_interact(mob/user, ui_key = "main", datum/tgui/ui = null, force_open = FALSE, datum/tgui/master_ui = null, datum/tgui_state/state = GLOB.tgui_admin_state)
+	ui = SStgui.try_update_ui(user, src, ui_key, ui, force_open)
+	if(!ui)
+		ui = new(user, src, ui_key, "ERTmanager", name, 350, 430, master_ui, state)
+		ui.autoupdate = TRUE
+		ui.open()
+
+/datum/tgui_module/ert_manager/tgui_data(mob/user)
+	var/list/data = list()
+	data["str_security_level"] = capitalize(get_security_level())
+	switch(GLOB.security_level)
+		if(SEC_LEVEL_GREEN)
+			data["security_level_color"] = "green";
+		if(SEC_LEVEL_BLUE)
+			data["security_level_color"] = "blue";
+		if(SEC_LEVEL_RED)
+			data["security_level_color"] = "red";
+		else
+			data["security_level_color"] = "purple";
+	data["ert_type"] = ert_type
+	data["com"] = commander_slots
+	data["sec"] = security_slots
+	data["med"] = medical_slots
+	data["eng"] = engineering_slots
+	data["jan"] = janitor_slots
+	data["par"] = paranormal_slots
+	data["cyb"] = cyborg_slots
+	data["total"] = commander_slots + security_slots + medical_slots + engineering_slots + janitor_slots + paranormal_slots + cyborg_slots
+	data["spawnpoints"] = GLOB.emergencyresponseteamspawn.len
+	return data
+
+/datum/tgui_module/ert_manager/tgui_act(action, params)
+	if(..())
+		return
+	. = TRUE
+	switch(action)
+		if("ert_type")
+			ert_type = params["ert_type"]
+		if("toggle_com")
+			commander_slots = commander_slots ? 0 : 1
```

Comment:
```suggestion
			commander_slots = !commander_slots
```

Reply:
This var is a count of slots, not a boolean. I could convert it into a boolean, but then it would be inconsistent with how all the other slots are handled, so I'd prefer not to do that.